### PR TITLE
Fix broken links in Markdown documents

### PR DESCRIPTION
**Pull request by pabrahamsson**
#### What is this PR About?
This will update a few broken links in Markdown documents.
**NOTE:** There are still two broken links in [s2i-liberty/README](/redhat-cop/containers-quickstarts/blob/master/build-s2i-liberty/README.md#considerations-on-http-session-failover) that I haven't found updated pages for.

I also plan to add a CI test for this that will address #304 

#### How do we test this?
```
podman run -it --rm -v $PWD:/code:z registry.access.redhat.com/ubi8/nodejs-12 bash -c 'npm install --save-dev markdown-link-check && find /code -type f -name "*.md" -exec markdown-link-check --quiet --config /code/linkcheck-config.json {} \;'
```

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,12 +54,15 @@ A set of buildConfigs for building custom Jenkins images for OpenShift.
 
 ### Gitlab Runners
 
-A set of [Gitlab Runners](./gitlab-runners/README.md) for your [Gitlab CI/CD](https://docs.gitlab.com/runner/).
+Gitlab Runners for yoru [Gitlab CI/CD](https://docs.gitlab.com/runner/).
```
Comment (garethahealy) [inline README.md:57]: yoru?

**Comment by garethahealy** [comment]
@pabrahamsson ; just 1 spelling mistake, but lgtm

File: README.md (hunk 1 above)
Comment (pabrahamsson) [reply README.md]: Thanks, updated.

**Pull request merged**